clevercsv/_optional.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+"""Code for dealing with optional dependencies
+
+The functionality in this file is largely based on similar functionality in the 
+Pandas library.
+
+Author: G.J.J van den Burg
+Copyright: 2020, The Alan Turing Institute
+License: See LICENSE file.
+
+"""
+
+import importlib
+
+VERSIONS = {
```

Comment:
This seems like is not used? I see the versions defined in the setup.py.

Reply:
Good catch, that will need to be updated.